tests/unit/lastuser_core/test_model_client_AuthToken.py

```diff
@@ -128,37 +128,37 @@ def test_authtoken_all(self):
         result1 = models.AuthToken.all(pottermania.owners.users)
         self.assertIsInstance(result1, list)
         self.assertIsInstance(result1[0], models.AuthToken)
-        self.assertItemsEqual(result1, [herminone_token])
+        self.assertCountEqual(result1, [herminone_token])
```

Comment:
Why are we comparing counts instead of lists? If there is a sorting problem, cast into sets first.

Reply:
`assertItemsEqual` in py2 is named `assertCountEqual` in py3 for some reason.